src/main/webapp/app/overview/visualizations/exercise-scores-chart/exercise-scores-chart.component.html

```diff
@@ -11,7 +11,7 @@ <h3>{{ 'artemisApp.exercise-scores-chart.title' | artemisTranslate }}</h3>
             </li>
             <li *ngFor="let type of typeSet">
                 <label>
-                    <input class="ms-2" (change)="toggleExerciseType(type)" [checked]="chartFilter.get(type.toLowerCase().replace('_', '-'))" type="checkbox" />
+                    <input class="ms-2" (change)="toggleType(type)" [checked]="chartFilter.get(type.toLowerCase().replace('_', '-'))" type="checkbox" />
```

Comment:
Found one more occurrence that could be replaced (right?) 😬

Reply:
Apparently I did not remove the file filter when searching for the command in all files. Thanks for the heads up. Changed in 20b96ce 👍